package.json

```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "lint": "./node_modules/eslint/bin/eslint.js ./*.js ./assets ./deserializers ./serializers ./services ./test",
     "test": "./node_modules/mocha/bin/mocha 'test/**/*.js'",
+    "test:int": "./node_modules/mocha/bin/mocha 'test/integration/**/*.js'",
+    "test:ci": "ENV=.env-test npm run test && ENV=.env-test-old-db npm run test:int",
```

Comment:
I still really don't like that and it's harder to know on which database you are. And if you want to add one  more version you need to change this command line every time.
We could do something like that: https://github.com/ForestAdmin/lumber/pull/318
It does a simple describe and it's pretty easy to use/extends
![Capture d’écran 2019-11-07 à 11 19 56](https://user-images.githubusercontent.com/3263749/68381150-6c596c80-0151-11ea-9da7-5a9e811e9383.png)